src/main/java/ua/com/javarush/gnew/bruteForce/FrequencyAnalysis.java

```diff
@@ -0,0 +1,80 @@
+package ua.com.javarush.gnew.bruteForce;
+
+import ua.com.javarush.gnew.crypto.EncryptionUtil;
+import ua.com.javarush.gnew.language.Language;
+import ua.com.javarush.gnew.languageDetector.CreateLanguage;
+import ua.com.javarush.gnew.languageDetector.LanguageDetector;
+
+import java.util.*;
+
+public class FrequencyAnalysis {
+    private int key_bf = 0;
+    private final DecryptionHelper decryptionHelper = new DecryptionHelper();
+    public int getKey_bf(){
+        return key_bf;
+    }
+
+    public String analysis(String encryptedText, String referenceText) {
```

Comment:
Very good approach, I like it.
This method also cad be simplified a bit to be more readable.